### DISCUSSION TITLE
关于 DOWNGRADE（降级）和LIMIT_SPEED的疑问

### DISCUSSION THREAD

**Discussion by ezjbc** [rustdesk/rustdesk#7532] [Q&A]
DOWNGRADE_THRESHOLD: 0.66
DOWNGRADE_START_CHECK: 1800s
LIMIT_SPEED: 4Mb/s
TOTAL_BANDWIDTH: 1024Mb/s
SINGLE_BANDWIDTH: 16Mb/s

问题1：DOWNGRADE（降级）是什么意思，是否意味着宽带负载达到66%，进行全员降级限速。
问题2：LIMIT_SPEED和SINGLE_BANDWIDTH之间都是限速，有什么区别吗？

**Comment by xlionjuan** [comment]
先看這個
https://github.com/rustdesk/rustdesk-server/discussions/371

Downgrade 不確定是什麼，我猜是 congestion control
I guess it is related to congestion control


Limit speed 是針對單一設備限速
For single device limit speed 

Single/total bandwidth, 前者是單一session 限速（包含檔案傳輸），後者是整個伺服器的限速
First is for single session(including file transfers), second is for whole server

**Reply by xlionjuan** [reply]
https://github.com/rustdesk/rustdesk-server/discussions/371#discussioncomment-8511523
